javascript_nodejs/5.multiturn-prompts-bot/README.md

```diff
@@ -0,0 +1,44 @@
+This sample shows how to use the prompts classes included in `botbuilder-dialogs`. This bot will ask for the user's name and age, then store the response. It demonstrates a multi-turn dialog flow using a text prompt, a number prompt, and multiple state accessors to store and retrieve values.
+
+# To try this sample
+- Clone the repository
+    ```bash
+    git clone https://github.com/microsoft/botbuilder-samples.git
+    ```
+- In a terminal, navigate to javascript_nodejs/5.multiturn-prompts-bot
+    ```bash
+    cd javascript_nodejs/5.multiturn-prompts-bot
+    ```
+    - Point to the MyGet feed 
+    ```bash
+    npm config set registry https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
+    ```
+- Install modules and start the bot
+    ```bash
+    npm i & npm start
+    ```
+- To reset registry, you can do
+    ```bash
+    npm config set registry https://registry.npmjs.org/
+    ```
+
+# Testing the bot using Bot Framework Emulator
+[Microsoft Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
```

Comment:
>(https://github.com/microsoft/botframework-emulator [](start = 34, length = 51)

Replace with the aka link (below).

Reply:
I'm not sure what you mean by this, @cleemullins

Reply:
Oh, the aka.ms shortened link.  Is there one that points to the root of the Github project? Do we want to continue pointing to this? The aka.ms/botframework-emulator link points to the releases.